help/C/gnucash-help.xml

```diff
@@ -310,6 +310,9 @@
 	<firstname>J. Alex</firstname> <surname>Aycinena</surname>
       </author>,
       <author>
+  <firstname>David</firstname> <surname>Cousens</surname>
```

Comment:
Indent, not out-dent. ;-)

Reply:
@jralls I wonder, if we should define and apply a (relaxed) style for our docbook documents, e.g. indent 2 spaces, linelenght 80

Reply:
@jralls I noticed when I opened gnucash-help.xml  to fix the above that all the other author entries showed up as being out-dented not indented. Is that worth fixing? I am using xed with an indent at 2 characters and tabs converted to spaces. If I open it in emacs they all display correctly as emacsand some are indented as tabs and some as spaces. All the <author> and </author> tags are indented with 6 spaces while most of the <firstname> tags are indented with tabs (which xed displays as 2 spaces - it doesn't change tabs to spaces on read but does during editing).  Most of the rest of the document is indented with spaces and not tab stops.

Reply:
David,
I don't know xed, but in emacs you can `M-< M-> M-X untabbify` to change all of the tabs to spaces, after which it should look right everywhere. Commit as a separate "whitespace" change.

Reply:
I'd encourage emacs 👍 which seems to do things right every time!